elide-async/src/test/java/com/yahoo/elide/async/operation/GraphQLTableExportOperationTest.java

```diff
@@ -138,6 +143,69 @@ public void testProcessBadQuery() throws URISyntaxException, IOException  {
         assertEquals("Bad Request Body'Can't parse query: { tableExport { edges { node { id principalName}  } }'", queryResultObj.getMessage());
     }
 
+    @Test
+    public void testProcessQueryWithRelationship() {
+        TableExport queryObj = new TableExport();
+        String query = "{\"query\":\"{ group { edges { node { name products {edges { node { name } } } } } } }\", \"variables\":null}";
+        String id = "edc4a871-dff2-4194-804e-d80075cf827d";
+        queryObj.setId(id);
+        queryObj.setQuery(query);
+        queryObj.setQueryType(QueryType.GRAPHQL_V1_0);
+        queryObj.setResultType(ResultType.CSV);
+
+        GraphQLTableExportOperation graphQLOperation = new GraphQLTableExportOperation(new JSONExportFormatter(elide),
+                        asyncExecutorService, queryObj, requestScope, engine);
+        TableExportResult queryResultObj = (TableExportResult) graphQLOperation.call();
+
+        assertEquals(200, queryResultObj.getHttpStatus());
+        assertEquals("Export is not supported for Query that requires traversing Relationships.",
+                        queryResultObj.getMessage());
+        assertEquals(null, queryResultObj.getRecordCount());
+        assertEquals(null, queryResultObj.getUrl());
+    }
+
+    @Test
+    public void testProcessQueryWithMultipleProjection() {
+        TableExport queryObj = new TableExport();
+        String query = "{\"query\":\"{ tableExport { edges { node { principalName } } } asyncQuery { edges { node { principalName } } } }\",\"variables\":null}";
+        String id = "edc4a871-dff2-4094-804e-d80075cf827d";
+        queryObj.setId(id);
+        queryObj.setQuery(query);
+        queryObj.setQueryType(QueryType.GRAPHQL_V1_0);
+        queryObj.setResultType(ResultType.CSV);
+
+        GraphQLTableExportOperation graphQLOperation = new GraphQLTableExportOperation(new JSONExportFormatter(elide),
+                        asyncExecutorService, queryObj, requestScope, engine);
+        TableExportResult queryResultObj = (TableExportResult) graphQLOperation.call();
+
+        assertEquals(200, queryResultObj.getHttpStatus());
+        assertEquals("Export is only supported for single Query with one root projection.",
+                        queryResultObj.getMessage());
+        assertEquals(null, queryResultObj.getRecordCount());
+        assertEquals(null, queryResultObj.getUrl());
+    }
+
+    @Test
+    public void testProcessMultipleQuery() {
```

Comment:
How is this test different than the one above it?

Reply:
this one has 2 queries with single projection and other one has single query with more than one projection